apps/aehttp/src/sc_ws_handler.erl

```diff
@@ -62,6 +89,12 @@ websocket_handle({text, MsgBin}, #handler{protocol = Protocol,
 websocket_handle(_Data, H) ->
     {ok, H}.
 
+websocket_info(?ERROR_TO_CLIENT(Err), #handler{protocol = Protocol} = H) ->
+    {reply, Resp} = sc_ws_api:error_response(Protocol, Err),
+    timer:sleep(1000),
```

Comment:
?

Reply:
This is when an initial openning settings were erronous - we give some time the client to open the connection on their side so the message is not lost

Reply:
This case seems rare enough that it should be logged using lager.

Reply:
Addressed